config/serialization.go

```diff
@@ -4,27 +4,22 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"log"
 	"os"
+	"os/user"
 	"path/filepath"
 	"reflect"
 
-	homedir "github.com/mitchellh/go-homedir"
 	yaml "gopkg.in/yaml.v2"
 )
 
 // configFile is the location of the configuration file
 func configFile() string {
-	configFolder := os.Getenv("XDG_CONFIG_HOME")
-	if configFolder == "" {
-		homeFolder := os.Getenv("HOME")
-		if homeFolder == "" {
-			homeFolder, _ = homedir.Dir()
-		}
-		if homeFolder != "" {
-			configFolder = filepath.Join(homeFolder, ".config")
-		}
+	u, err := user.Current()
+	if err != nil {
+		log.Fatalf("Cannot determine user home directory required for storing taskcluster-cli configuration file: %v", err)
 	}
-	return filepath.Join(configFolder, "taskcluster.yml")
+	return filepath.Join(u.HomeDir, "taskcluster.yml")
 }
 
```

Comment:
As well, from https://github.com/mitchellh/go-homedir: 

> **Why not just use `os/user`?** The built-in `os/user` package requires
cgo on Darwin systems. This means that any Go code that uses that package
cannot cross compile. But 99% of the time the use for `os/user` is just to
retrieve the home directory, which we can do for the current user without
cgo. This library does that, enabling cross-compilation.